lib/activerecord-mysql-unsigned/active_record/v4/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -4,42 +4,34 @@ module ActiveRecord
   module ConnectionAdapters
     class AbstractMysqlAdapter < AbstractAdapter
  
-      class ChangeColumnDefinition < Struct.new(:column, :type, :options) #:nodoc:
+      class ChangeColumnDefinition < Struct.new(:column, :name) #:nodoc:
       end
```

Comment:
Applying this change may cause same problem before v4.2.0, I think.  ( see rails/rails@3c10043 )
(But test with ar4.1 or ar4.0 works.)
I think need to cut out v42 over version class.